### PR TITLE
Fix/minor improvements

### PR DESCRIPTION
**Pull request by Tithugues**
### Description

This MR includes many improvements on many topics:
* merge isset / unset
* use constants in some places
* use relevant static calls when needed

### CODE REVIEW

File: libraries/classes/Config.php [hunk 1]
```diff
@@ -439,7 +439,7 @@ public function checkGitRevision(): void
         }
 
         if ($common_dir_contents = @file_get_contents($git_folder . '/commondir')) {
-            $git_folder = $git_folder . DIRECTORY_SEPARATOR . trim($common_dir_contents);
+            $git_folder .= trim($common_dir_contents) . DIRECTORY_SEPARATOR;
```
Comment (williamdes) [inline libraries/classes/Config.php]: You inverted the operation

File: libraries/classes/Export.php [hunk 2]
```diff
@@ -589,7 +589,7 @@ public function exportServer(
         string $separate_files
     ): void {
         if (! empty($db_select)) {
-            $tmp_select = implode($db_select, '|');
+            $tmp_select = implode('|', $db_select);
```
Comment (williamdes) [inline libraries/classes/Export.php]: Good catch!

**Review by williamdes — CHANGES_REQUESTED** [review]
LGTM
I left one comment

File: libraries/classes/Config.php (hunk 1 above)
Comment (Tithugues) [reply libraries/classes/Config.php]: You're right, sorry for that.
This should be fixed now.

**Review by MauricioFauth — APPROVED** [review]

File: libraries/classes/Util.php [hunk 3]
```diff
@@ -3652,7 +3651,7 @@ public static function getGISFunctions(
             'type' => 'int',
         ];
 
-        $geom_type = trim(mb_strtolower((string) $geom_type));
+        $geom_type = mb_strtolower(trim((string)$geom_type));
```
Comment (mauriciofauth) [inline libraries/classes/Util.php]: The coding standard requires 1 space after the cast statement.

```suggestion
        $geom_type = mb_strtolower(trim((string) $geom_type));
```

**Review by williamdes — APPROVED** [review]

File: libraries/classes/Util.php (hunk 3 above)
Comment (Tithugues) [reply libraries/classes/Util.php]: Sorry. Fixed.

**Comment by mauriciofauth** [comment]
Merged in ffdc2cf7bf66b1e7f1090b9d05794483bc3f4df0. Thanks for your contribution.

**Pull request closed without merging**